core/application-crypto/src/lib.rs

```diff
@@ -53,9 +53,9 @@ pub use traits::*;
 #[macro_export]
 macro_rules! app_crypto {
 	($module:ident, $key_type:expr) => {
-		#[cfg(feature="std")]
+		#[cfg(any(feature = "full_crypto", feature = "std"))]
```

Comment:
This will not work. When you for example want to use the app crypto from grandpa in your SGX env, the code will not be accessible.

Reply:
that's true. It won't be a problem for us now, but if we one day want to run the entire client in SGX (or some other no_std env) this will be an issue

Reply:
...discussion continued below